content/livesync/quickstart.textile

```diff
@@ -1,7 +1,228 @@
 ---
 title: Quickstart
-meta_description: "."
+meta_description: "A quickstart guide to learn the basics of integrating the Ably LiveSync product into your application."
 product: livesync
 languages:
   - javascript
 ---
+
+LiveSync facilitates broadcasting realtime updates from backend databases to frontend clients at scale. Leveraging Ably's Pub/Sub Channels, LiveSync ensures that data updates are propagated reliably and in order to all connected clients in realtime.
+
+This quickstart will provide a brief introduction into the LiveSync components integrated into a realtime web application enabling people to comment on a post. Following this, you'll follow instructions to get this demo application running locally.
```

Comment:
I do think that we should not only let them clone and run the project but also we should explain the folder structure briefly so they know what's happening where before we start delving into specific snippets from specific files. I think we should tell them the overall working architecture of the app so they have an idea.